docs/content/foundations/utilities-overview.mdx

```diff
@@ -0,0 +1,189 @@
+
+For a complete list of the utilities and their bases, see each utility’s documentation page, or the Modules table, below.
+
+The utility values are drawn from your project theme-settings tokens but can be extended to include additional values drawn from our palette of common units (like px, em, ch, and percentages) as well as the broader USWDS system palette of colors, grid spacing, and font scale. (Each utility’s Advanced settings section provides more information on customization.)
+
+| USWDS Utility        | Sass                                        |
+| :------------------- | :------------------------------------------ |
+| `.bg-primary-darker` | `background-color: color('primary-darker')` |
+| `.bg-red-50v`        | `background-color: color('red-50v')`        |
+| `.text-underline`    | `text-decoration: underline`                |
+| `.margin-x-auto`     | `margin-left-auto`<br/>`margin-right-auto`  |
+| `.padding-bottom-1`  | `padding-bottom: units(1)`                  |
+
+### Utility Class Modifiers
+
+Utility modules can be configured to output additional CSS classes for targeting specific breakpoints or the link-related pseudo-classes. Each utility module under [Foundations](/foundations) documents which modifiers are enabled by default.
+
+<h4 id="responsive">Responsive Modifiers</h4>
+
+Add a responsive breakpoint prefix separated with a `:` to target a utility at a responsive breakpoint and higher, following a mobile-first methodology.
+
+**Example**
+
+`.tablet:padding-y-2`
+
+**Output**
+
+```
+@media screen and (min-width: 640px) {
+	.tablet\:padding-y-2 {
+		padding-top: 1rem;
+		padding-bottom: 1rem;
+	}
+}
+```
+
+Set up which breakpoints are available to USWDS utilities by setting the `$theme-utility-breakpoints` variable in your USWDS settings configuration.
+
+| Breakpoint Name | Width  | Default |
+| :-------------- | :----- | :------ |
+| card            | 160px  | false   |
+| card-lg         | 240px  | false   |
+| mobile          | 320px  | true    |
+| mobile-lg       | 480px  | true    |
+| tablet          | 640px  | true    |
+| tablet-lg       | 880px  | true    |
+| desktop         | 1024px | true    |
+| desktop-lg      | 1200px | false   |
+| widescreen      | 1400px | true    |
+
+Set whether a utility family outputs with responsive variants by setting the value of `$[utility_family]-settings.responsive` in your USWDS settings configuration. When modifying settings stored as a map, undefined map elements will keep their default values.
+
+**Example**
+
+```
+@use "uswds-core" with (
+  $background-color-settings: (
+    responsive: true
+  )
+);
+```
+
+<h4 id="state">State Modifiers</h4>
```

Comment:
When we link to this section from the utility-info-table, we have a hyperlink of say, `Active`. We should really make the text in this column of the table all lower case. This way people know which "state prefixes" they can use. Otherwise this should spell it out for them.